### PR TITLE
updates for 2022.1

### PR DESCRIPTION
**Pull request by xingyz**
updating the sample dialect in the root folder. 
* added new properties introduced in 2021.3 and 2022.1
* removed some description to "declutter" the file, and put them into wiki pages.

Also added 9 pages for Wiki https://github.com/Esri/insights-connector-guide/wiki. All the pages titled "Configure xxx" are new

### CODE REVIEW

**Comment by fran8806** [comment]
@xingyz We need to update the year of copyright statement to 2022. Both dialect.yml and properties.yml have it.

**Review by fran8806 — APPROVED** [review]

File: dialect.yml [hunk 1]
```diff
@@ -15,14 +15,14 @@
 # 
 
 
-# id of the connector type (mandatory)
+# Id of the connector type (mandatory)
```
Comment (vcove) [inline dialect.yml:18]: ID

File: dialect.yml [hunk 2]
```diff
@@ -15,14 +15,14 @@
...
 id: sample.dialect
 
 
-# id of the parent dialect. It allows your dialect to inherit all unconfigured properties from a parent dialect.
+# Id of the parent dialect. It allows your dialect to inherit all unconfigured properties from a parent dialect.
```
Comment (vcove) [inline dialect.yml:22]: ID

**Review by vcove — APPROVED** [review]
Approved pending final change to ID

**Comment by xingyz** [comment]
@vcove I've made changes to use "ID" instead of "Id". 
Merging this PR now.

**Pull request merged**